src/modules/cpu/kernel/crop_mirror_normalize.hpp

```diff
@@ -131,9 +132,9 @@ RppStatus crop_mirror_normalize_u8_u8_host_tensor(Rpp8u *srcPtr,
                     for (; vectorLoopCount < bufferLength; vectorLoopCount += 3)
                     {
```

Comment:
for all for loops move the bracket to previous line

Reply:
addressed in code walkthrough call. Common change in a separate PR after these PR merges.